packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.service.js

```diff
@@ -398,23 +398,30 @@ export default class {
     );
   }
 
-  deleteZertoOptionOrderInUserPref(drpKey) {
-    return this.ovhUserPref.remove(drpKey, true);
+  deleteZertoOptionOrderInUserPref(pccId) {
+    const preferenceKey = this.constructor.formatPreferenceKey(
+      pccId,
+      DEDICATEDCLOUD_DATACENTER_ZERTO.title,
+    );
+
+    return this.ovhUserPref.remove(preferenceKey, true);
   }
 
   /* ------- Order ZERTO option ------ */
 
   disableDrp(drpInformations) {
-    const deleteDrpPromise =
-      drpInformations.drpType === DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS.ovh
-        ? this.disableDrpOvh(drpInformations)
-        : this.disableDrpOnPremise(drpInformations);
-
     const drpPccId = drpInformations.primaryPcc.serviceName;
 
-    return deleteDrpPromise
-      .then(() => this.deleteZertoOptionOrderInUserPref(drpPccId))
-      .then(() => this.deleteDisableSuccessAlertPreference(drpPccId));
+    return (drpInformations.drpType ===
+    DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS.ovh
```

Comment:
Is this the new `Prettier` indentation ? 🤔

Reply:
Yes it seems

Reply:
Well it's horrible 😅